operator/apis/platform/v1alpha1/central_types.go

```diff
@@ -293,6 +293,10 @@ type ExposureRoute struct {
 	//+kubebuilder:default=false
 	//+operator-sdk:csv:customresourcedefinitions:type=spec,order=1
 	Enabled *bool `json:"enabled,omitempty"`
+
+	// Specify a custom hostname for the central route.
+	//+operator-sdk:csv:customresourcedefinitions:type=spec,order=2
```

Comment:
```suggestion
	// Specify a custom hostname for the central route.
	// If unspecified, an appropriate default value will be automatically chosen by OpenShift route operator.
	//+operator-sdk:csv:customresourcedefinitions:type=spec,order=2
```
Just to stress that this is optional.